service/cloudconfig/v_0_1_0_master_template.go

```diff
@@ -137,6 +137,12 @@ func (e *v_0_1_0MasterExtension) Files() ([]k8scloudconfig.FileAsset, error) {
 			Owner:        FileOwner,
 			Permissions:  FilePermission,
 		},
+		{
```

Comment:
Also please put there comment `etcd_data_dir drop-in` so we have logical division :)